src/Raven.Server/Documents/Replication/ReplicationItems/DocumentReplicationItem.cs

```diff
@@ -21,7 +21,7 @@ public sealed class DocumentReplicationItem : ReplicationBatchItem
         public override DynamicJsonValue ToDebugJson()
         {
             var djv = base.ToDebugJson();
-            djv[nameof(Collection)] = Collection?.ToString(CultureInfo.InvariantCulture) ?? Constants.Documents.Collections.EmptyCollection;
+            djv[nameof(Collection)] = Collection?.ToString(CultureInfo.InvariantCulture) ?? CollectionName.GetCollectionName(Data);
```

Comment:
what about the rest of the items?

Reply:
because we don't write the `Collection` directly only for `DocumentReplicationItem` (in `Write` method)